test/python/WMCore_t/Services_t/Rucio_t/Rucio_t.py

```diff
@@ -237,7 +237,26 @@ def testGetPFN(self):
         """
         Test `getPFN` method
         """
-        self.assertRaises(NotImplementedError, self.myRucio.getPFN)
+        # simplest call
+        resp = self.client.getPFN(site="T2_CH_CERN", lfns="/store")
```

Comment:
You're actually using the wrong object here. Please replace `self.client` by `self.myRucio` in all these calls below.

Reply:
I had `self.myRucio` before and got the `getPFN() takes at least 3 arguments (1 given)`   I am quite confused.
Anyhow I am doing as you suggest.  Let me also comment all my additional things and only leave your example. I need to work on CRAB code.